src/webhook/dto/custom-class-validators.ts

```diff
@@ -0,0 +1,40 @@
+import { registerDecorator, ValidationOptions } from 'class-validator';
+import { WebhookEventType } from './webhook-event-types';
+
+export function NoDuplicateAndOtherTypes(
```

Comment:
If you follow [this comment](https://github.com/KubaBaniak/RecipeAppNestjs/pull/64/files#r1360468713), then you will end up removing this decorator.